### ISSUE TITLE
No battery status

### ISSUE THREAD

**Issue by infinisil** [kellyjonbrazil/jc#393]
The `.status` field is missing in the resulting JSON: 
```
$ acpi | tee /dev/stderr | jc --acpi
Battery 0: Not charging, 100%
[{"type":"Battery","id":0}]
```

It seems like the string `Not charging` is not detected properly, it comes from
```
$ cat /sys/class/power_supply/BAT1/status
Not charging
```

**Comment by kellyjonbrazil** [comment]
Hi - thanks for reporting this issue. Could you add the complete `acpi` output to this thread and I'll take a look?

Thanks!

**Comment by infinisil** [comment]
I guess the `tee` up there is a bit obscure, but the `acpi` output is
```
Battery 0: Not charging, 100%
```

:smile:

**Comment by kellyjonbrazil** [comment]
Sorry - yeah I got that a little late. :) I have a fix in `dev` you can test by copying the new file to the `jc` plugins folder:

https://github.com/kellyjonbrazil/jc/blob/dev/jc/parsers/acpi.py

- Linux/unix: `$HOME/.local/share/jc/jcparsers`
- macOS: `$HOME/Library/Application Support/jc/jcparsers`
- Windows: `$LOCALAPPDATA\jc\jc\jcparsers`

Let me know if you run into any issues, thanks!

**Comment by Lidahatami** [comment]
<details><summary>Details</summary>
<p>



</p>
</details>

**Comment by kellyjonbrazil** [comment]
Released in v1.23.2
